bodhi/server/models.py

```diff
@@ -982,6 +982,34 @@ def setting_status(self) -> typing.Optional[str]:
         """
         return config.get(f'{self.setting_prefix}.status', None)
 
+    def get_testing_side_tag(self, from_tag: str) -> str:
+        """
+        Return the testing side tag for this ``Release``.
+
+        Args:
+            from_tag: Name of side tag from which ``Update`` was created.
+
+        Returns:
+            Testing side tag used in koji.
+        """
+        side_tag_postfix = config.get(
+            '{}.koji-testing-side-tag'.format(self.setting_prefix), "-testing")
```

Comment:
should we use f-strings here ?

Reply:
Do we have some standard here? I saw `.format()` in some places, `%s` in others and I also saw f-string somewhere.

Reply:
we are slowly moving to f-strings, I think all new code should be using f-string.

Reply:
Fixed